lib/GenomeFileUtil/core/MiscUtils.py

```diff
@@ -0,0 +1,16 @@
+"""
+This file contains miscellaneous utilities that are not tied to the concept of a genome.
+"""
+
+
+def validate_lists_have_same_elements(l1, l2):
```

Comment:
Why did you duplicate this in two places?

Reply:
no good reason, removing the function is GenomeUtils. Looking to declutter the repo.